src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -255,6 +255,13 @@ enum class ConfigType : uint16_t {
    */
   USER_DEVICE_ID = 301,
 
+  /**
+   * A bitmask indicating which profiling features are enabled.
+   *
+   * Payload format: `uint8_t` (see @ref sat_type_masks)
+   */
+  PROFILING_ENABLED = 310,
```

Comment:
Maybe `PROFILING_MASK` instead? Enabled sort of implies a single bool instead of multiple things